### ISSUE TITLE
We love screenshots!

### ISSUE THREAD

**Issue by GreenLunar** [asciimoo/omnom#14]

**Comment by asciimoo** [comment]
Ohi, thanks! ❤️ 
And.. its even better, omnom creates snapshots instead of screenshots which are self contained, searchable html documents with inlined images, css, fonts, clickable links, submitable forms, etc.. =]

**Comment by GreenLunar** [comment]
I mean, the only preview I have is this 
![image](https://user-images.githubusercontent.com/2053884/152889335-9515669a-3eae-4625-ba58-0c4fbad0cbc9.png)
it would be nice to have some more to get some sense of the expected experience.

**Comment by asciimoo** [comment]
Indeed. It's on my todo list, but the project is still in an early development phase, so it has some bigger shortcomings we need to address before. =]
